python/ray/tune/integration/comet.py

```diff
@@ -199,6 +209,33 @@ def log_trial_result(self, iteration: int, trial: "Trial", result: Dict):
         for k, v in episode_logs.items():
             experiment.log_curve(k, x=range(len(v)), y=v, step=step)
 
+    def log_trial_save(self, trial: "Trial"):
+        if self.save_checkpoints and trial.checkpoint:
+            if trial not in self._trial_experiments:
+                self.log_trial_start(trial)
```

Comment:
Is this something that is required? Seems like we're doing this in `log_trial_result` as well.

Out of scope for this PR, but this logic seems like it shouldn't be the responsibility of the callback. Shouldn't the trial scheduler make sure that the trial is started before trying to log to it?